react/features/feedback/components/FeedbackDialog.web.js

```diff
@@ -0,0 +1,342 @@
+        this._onMessageChange = this._onMessageChange.bind(this);
+        this._onScoreContainerMouseLeave
+            = this._onScoreContainerMouseLeave.bind(this);
+        this._onSubmit = this._onSubmit.bind(this);
+    }
+
+    /**
+     * Emits an analytics event to notify feedback has been opened.
+     *
+     * @inheritdoc
+     */
+    componentDidMount() {
+        JitsiMeetJS.analytics.sendEvent('feedback.open');
+    }
+
+    /**
+     * Invokes the onClose callback, if defined, to notify of the close event.
+     *
+     * @inheritdoc
+     */
+    componentWillUnmount() {
+        if (this.props.onClose) {
+            this.props.onClose();
+        }
+    }
+
+    /**
+     * Implements React's {@link Component#render()}.
+     *
+     * @inheritdoc
+     * @returns {ReactElement}
+     */
+    render() {
+        const { message, mousedOverScore, score } = this.state;
+        const scoreToDisplayAsSelected
+            = mousedOverScore > -1 ? mousedOverScore : score;
+
+        const scoreIcons = this._scoreClickConfigurations.map(
+            (config, index) => {
+                const isFilled = index <= scoreToDisplayAsSelected;
+                const activeClass = isFilled ? 'active' : '';
+                const className
+                    = `star-btn ${scoreAnimationClass} ${activeClass}`;
+
+                return (
+                    <a
+                        className = { className }
+                        key = { index }
+                        onClick = { config._onClick }
+                        onMouseOver = { config._onMouseOver }>
+                        { isFilled
+                            ? <StarFilledIcon
+                                label = 'star-filled'
+                                size = 'xlarge' />
+                            : <StarIcon
+                                label = 'star'
+                                size = 'xlarge' /> }
+                    </a>
+                );
+            });
+
+        const { t } = this.props;
+
+        return (
+            <Dialog
+                okTitleKey = 'dialog.Submit'
+                onCancel = { this._onCancel }
+                onSubmit = { this._onSubmit }
+                titleKey = 'feedback.rateExperience'>
+                <div className = 'feedback-dialog'>
+                    <div className = 'rating'>
+                        <div className = 'star-label'>
+                            <p id = 'starLabel'>
+                                { t(SCORES[scoreToDisplayAsSelected]) }
+                            </p>
+                        </div>
+                        <div
+                            className = 'stars'
+                            onMouseLeave = { this._onScoreContainerMouseLeave }>
+                            { scoreIcons }
+                        </div>
+                    </div>
+                    <div className = 'details'>
+                        <textarea
```

Comment:
https://bitbucket.org/atlassian/atlaskit/issues/15/how-to-create-a-multi-line-text-area-input